quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -273,42 +334,27 @@ impl Handler<DeleteSourceRequest> for ControlPlane {
             index_uid,
             source_id: request.source_id,
         };
-        // TODO: Notify indexing controller.
+
```

Comment:
I think we need to call ` self.ingest_controller.delete_source(&index_uid, &source_id).await;` here.

Reply:
Yes added.